aws/resource_aws_s3_bucket_test.go

```diff
@@ -3767,6 +3859,28 @@ resource "aws_s3_bucket" "destination" {
 `, randInt, randInt, randInt)
 }
 
+func testAccAWSS3BucketReplicationConfig_iamPolicy(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_iam_role" "test" {
+  name = %[1]q
```

Comment:
IAM role name matches [sweeper prefixes](https://github.com/terraform-providers/terraform-provider-aws/blob/8d319bdb8719ef1ff5de60fe9b230b93f3a1f777/aws/resource_aws_iam_role_test.go#L52-L60) by virtue of `tf-test-bucket-` prefix on bucket names.